src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Everything/EverythingOperationContinuationToken.cs

```diff
@@ -0,0 +1,44 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Text.Json;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Everything
+{
+    public class EverythingOperationContinuationToken
+    {
+        public EverythingOperationContinuationToken(int phase, string internalContinuationToken)
+        {
+            Phase = phase;
+            InternalContinuationToken = internalContinuationToken;
+        }
+
+        public int Phase { get; internal set; }
```

Comment:
>internal [](http://example.com/codeflow?start=32&length=8)

would it work with `private set` instead of `internal set`?

Reply:
`Phase` needs `internal set` because it is set in `PatientEverythingService` as well. `InternalContinuationToken` could be `private set` and I've fixed it in the update. Thanks for pointing this out.

Reply:
I've check code I don't see why can't you just do 
```
var phase =  token.Phase;
switch(phase){
...
}
```
I just personally prefer readonly fields because they save you from mutatable state mistakes, but it's up to you.

Reply:
Agree. I've set `Phase` to private in the update.